.github/workflows/publish_alpha.yml

```diff
@@ -34,6 +34,20 @@ jobs:
     runs-on: ubuntu-latest
     needs: update_version
     steps:
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
```

Comment:
This action is unmaintained and hasn't been updated in 3 years. Here's one they recommend in the README (try v2): https://github.com/softprops/action-gh-release